routes/articles.js

```diff
@@ -0,0 +1,45 @@
+const apiArticles = require("express").Router();
+const {
+  
+  getArticlesWithCommentCount,
+  getArticleById,
+} = require("../models/articles");
+
+const express = require("express");
+const router = express.Router();
+
+router.get("/", (req, res) => {
+  return getArticlesWithCommentCount()
+    .then((sortedArticlesDesc) => {
+      res.status(200).send({ articles: sortedArticlesDesc });
+    })
+    .catch((err) => {
```

Comment:
All catch blocks need changing to error handlers and allowing for errors other than 500, what if its an error than requires a different code or message?